src/main/java/com/exasol/adapter/response/converter/ResponseJsonConverter.java

```diff
@@ -0,0 +1,56 @@
+package com.exasol.adapter.response.converter;
+
+import com.exasol.adapter.json.SchemaMetadataSerializer;
+import com.exasol.adapter.response.CreateVirtualSchemaResponse;
+import com.exasol.adapter.response.DropVirtualSchemaResponse;
+import com.exasol.utils.JsonHelper;
+
+import javax.json.Json;
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+
+/**
+ * Converts response into Json format
```

Comment:
"... JSON format ..."
Please correct in other JavaDoc too.

Reply:
fixed